layers/+email/mu4e/packages.el

```diff
@@ -39,9 +39,28 @@
                 'ivy-completing-read
               'helm-completing-read))
 
+      (when mu4e-async
+        (require 'smtpmail-async)
+        (mu4e/set-send-function 'async-smtpmail-send-it))
+
+      (when (fboundp 'imagemagick-register-types)
+        (imagemagick-register-types))
```

Comment:
Indeed it is: http://www.djcbsoftware.nl/code/mu/mu4e/Viewing-images-inline.html